pkgs/applications/graphics/displaycal/default.nix

```diff
@@ -1,14 +1,16 @@
-{buildPythonPackage, stdenv, fetchurl, pkgconfig
- , libXext, libXxf86vm, libX11, libXrandr, libXinerama, libXScrnSaver
- , argyllcms, wxPython, numpy
- }:
-buildPythonPackage {
-  name = "displaycal-3.5.0.0";
+{ buildPythonPackage, stdenv, fetchurl, pkgconfig
+, libXext, libXxf86vm, libX11, libXrandr, libXinerama, libXScrnSaver
+, argyllcms, wxPython, numpy
+}:
+
+buildPythonPackage rec {
+  name = "DisplayCAL-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed. Just add `pname` and `version`.

Reply:
I keep forgetting that one, thanks!